src/components/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -77,6 +78,19 @@ export const Controls: Story = {
     },
     jest: ["RadioGroup.test.tsx"],
   },
+  play: async ({ canvasElement }) => {
+    const screen = within(canvasElement);
+
+    expect(screen.getByRole("radiogroup")).toBeInTheDocument();
+
+    await userEvent.click(screen.getByLabelText("Radio 3"));
+    await userEvent.click(screen.getByLabelText("Radio 2"));
+    await userEvent.click(screen.getByLabelText("Radio 5"));
+    await userEvent.keyboard("{arrowdown}");
+    await userEvent.keyboard("{arrowleft}");
+    await userEvent.keyboard("{arrowup}");
+    await userEvent.keyboard("{arrowright}");
```

Comment:
This code executes events, but shouldn't there also be tests that verify the outcome of the events?

Reply:
I thought the same. I was a bit confused but the Storybook doc examples don't always have expectations https://storybook.js.org/docs/writing-tests/interaction-testing

The actions and interactions _are_ the tests. When I did expectations, such as `  expect(screen.getByRole("radiogroup")).toBeInTheDocument();` above, they don't show up in the addon panel. So we can add more expectations but what is shown to users are the interactions.

Reply:
What happens if you add an expectation and it fails?

Reply:
The error will be shown. It's not very intuitive but it's an implicit success, so you're not told anything until something throws an error.

Reply:
Do you think this will ever replace the unit tests?

Reply:
No, it should not. I think there's a way to "share" tests between unit tests and interaction tests but I haven't looked too deep into it.

Reply:
I ask because I feel an expectation should be added whenever an userEvent is executed.  We do that with the unit tests.  Should we also do that here as well.  We can add all the click events that we want, but if nothing changes after the event and we don't explicitly test to verify that it has changed, does the fact that we executed a click event mean anything?

Reply:
@EdwinGuzman Looking back at the article about implicit assertions, none of the examples talk about testing user events.  They are all related to values and formats, but not to an event that may cause something to happen or change.  With the interective tests, we are triggering events, but can there always be an implicit assertion that something on screen or in the code has changed simply because an event was triggered?

Reply:
Right, I'd say yes and no and generally depends on the component we are testing and what we are testing for. If we want to test tabbing and moving arrows around, we visually see that new values get selected so it's working. It doesn't hurt to add expectations though.

I can add some expectations but because of how the addons display the tests, it won't appear in the list of interactions.

Reply:
> I can add some expectations but because of how the addons display the tests, it won't appear in the list of interactions.

I think that is ok.  They won't verify success, but they will throw errors if there is a problem.